packages/database/src/prisma-api.ts

```diff
@@ -404,7 +404,7 @@ export function getSdk(
       const purchases = userId
         ? await ctx.prisma.purchase.findMany({
             orderBy: {
-              createdAt: 'asc',
+              createdAt: 'desc',
```

Comment:
@jbranchaud this might have effects in other places but the problem was that with Epic React specifically we have some charges that aren't in this strip account, they are in another stripe account and we often have multiple team purchases because of age of the site and we grab `[0]` which fails

this sort the opposite way and lets us grab the first fresh team purchase